lib/main_common.pm

```diff
@@ -1869,6 +1870,19 @@ sub load_extra_tests_wicked {
     }
 }
 
+sub load_wicked_create_hdd {
```

Comment:
I think it make more sense to put it into `sle/main.pm` . in o3 we still using "normal"  `create_hdd_textmode` and I am not sure if we will have enough courage to change this one day ...

Reply:
oh, I just added it to product opensuse